pages/dkp/kommander/1.4/workspaces/edit-delete-workspaces/index.md

```diff
@@ -25,3 +21,19 @@ From the workspace selection dropdown at the top of the screen, select the works
 When creating or editing a workspace, you can use the **Advanced Options** to add, edit, or delete annotations and labels to your workspace. Both the annotations and labels are applied to the workspace namespace.
 
 Labels that are added to a workspace, are also applied to all the clusters in the workspace.
+
+## Delete workspaces
+
+<p class="message--note"><strong>NOTE: </strong>
+  Workspaces can only be deleted if all the clusters in the workspace have been deleted.
```

Comment:
```suggestion
  Workspaces can only be deleted if all the clusters in the workspace have been detached.
```

Reply:
Updated comment to include all the scenarios.